pico_w/bt/standalone/client.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include <stdio.h>
+#include <hci_dump_embedded_stdout.h>
```

Comment:
did you mean to include this? if so, perhaps " for consitency

Reply:
No. Now removed